tests/ImageSharp.Tests/Drawing/Text/TextGraphicsOptionsTests.cs

```diff
@@ -9,25 +11,139 @@ namespace SixLabors.ImageSharp.Tests.Drawing.Text
 {
     public class TextGraphicsOptionsTests
     {
+        private readonly TextGraphicsOptions newTextGraphicsOptions = new TextGraphicsOptions();
+        private readonly TextGraphicsOptions defaultTextGraphicsOptions = TextGraphicsOptions.Default;
+        private readonly TextGraphicsOptions cloneTextGraphicsOptions = TextGraphicsOptions.Default.Clone();
```

Comment:
We need proper tests for `Clone()`, which are actually altering the state of the instance before cloning it.

Reply:
Done.